bebraven/platform
# Allow adding lessons from assignment placement

Task: https://app.asana.com/0/1174274412967132/1186098743228538/f

I did this with as few changes as possible, because I'd rather not spend time moving controller stuff around until we have an actual design for this. 

To test:
1. Make sure the assignment placement link in your developer key is pointing to your /lti/assignment_selection/new
2. Add a new assignment
3. Continue as usual

<img width="461" alt="Screen Shot 2020-07-27 at 12 11 17 PM" src="https://user-images.githubusercontent.com/1382374/88570802-4c7d1800-d002-11ea-8c84-fba4eda274a6.png">
